jetty-server/src/test/java/org/eclipse/jetty/server/RequestTest.java

```diff
@@ -32,13 +32,16 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.sql.Array;
```

Comment:
java.sql? :)

Reply:
Fixed thanks, I don't have my normal cleanup rules in this project sorry.